src/Extensions/Extensions.csproj

```diff
@@ -16,9 +16,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\WebForms\WebForms.csproj" />
+     <ProjectReference Include="..\Compiler.Dynamic\Compiler.Dynamic.csproj" />
```

Comment:
Extensions probably shouldn't rely on the dynamic compiler

Reply:
We can tackle that in a different issue, though

Reply:
Created #44 to track that

Reply:
@twsouthwick else how below would work ? The PageCompilationOptions is part of Compiler.Dynamic

```C#
 public static ISystemWebAdapterBuilder AddWebFormsExtensions(this ISystemWebAdapterBuilder builder)
 {
     builder.Services.AddOptions<PageCompilationOptions>()
       .Configure(options =>
       {
           options.AddTypeNamespace<ScriptManager>("asp");
           options.AddTypeNamespace<ListView>("asp");
       });
     return builder;
 }
```

Reply:
not sure - that's why I created a new issue so we can get this in without worrying about it. We'll probably need a extensions.dynamic project or something like that - but we can figure that out later.

ideally, controls should be a lower layer than any compiler related thing (not totally possible due to webforms design, but it would be nice)